build.gradle

```diff
@@ -13,7 +13,7 @@ project.docsDirName='../docs/0.3.x/'
 
 configurations.all { resolutionStrategy.cacheChangingModulesFor 0, 'seconds' }
 
-version = '0.3.6.3'
+version = '0.3.6.4'
```

Comment:
Could we make the same changes targeting the `release-0.3.x` branch too? 

I imagine we'll eventually want to update snooze (and any other services) to use a newer version of inversion, and I'd love to have these bug fixes in the newer versions too.

I'd love to say we should update the 0.4.x and higher branches, but I think those have a lot of breaking changes compared to 0.3.x, so it may not make sense to spend the time on those updates too

Reply:
Why not go to the latest on 0.3.x? I think we're on 0.3.12+

Reply:
https://github.com/RocketPartners/liftck_heartbeat/blob/develop/microservice/build.gradle#L140

Reply:
I thought we only had inversion in snooze and in some NGRP projects, but it seems like we have it in a few places. Snooze currently uses 0.3.6.3, but release 0.3.x is up to 0.3.17. I'm not sure why snooze is running the older version, and the git history of 0.3.x and 0.3.6.x is different enough that I don't think I can just upgrade snooze to 0.3.17 without potential issues. I'll talk with Mason to see if he can give me some background on the git history for that repo

Reply:
After talking with Mason & Connor, who were the last people to work on 0.3.6.x, it seems like there are changes in further minor versions of 0.3.x that they couldn't test properly in snooze, so snooze has to run on an 0.3.6.x version for the time being. 

I think the best course of action here is to merge this change into 0.3.6.x to fix our immediate issue with the portal, which is actively causing change logs to be lost in prod, and I'll write up a ticket to get snooze onto the latest version of inversion's 0.3.x branch (while also integrating the changes in 0.3.6.x that aren't in 0.3.x yet) afterwards. 
@johnbartz lmk if that sounds good to you.